.travis.yml

```diff
@@ -8,8 +8,8 @@ cache: bundler
 
 rvm:
   - 2.0
-  - 2.1
-  - 2.2
+  - 2.1.10
+  - 2.2.10
```

Comment:
Could you leave this change out?

This will pin it to those minor versions while the old would pin to the latest minor version.

Reply:
You'd think that leaving them at the feature releases would do the right thing with RVM like it does on a desktop, but Travis does some weird things in RVM-land. We've opened bugs with Travis on this in the past, but they like to hold back those aliases to really old versions in their infrastructure. Here's an example recently in this repo where 2.2 mapped to 2.2.7 instead of 2.2.10:

```
4.94s$ rvm use 2.2 --install --binary --fuzzy
Using /home/travis/.rvm/gems/ruby-2.2.7
```

They VERY rarely update RVM on their containers / GCE images to pull in recent versions. We've worked around this by pinning to the full version so that RVM will go and fetch the latest. Since there will never be another 2.1 or 2.2 release this is pretty safe here.

Reply:
Oh. Glad to know. =)